.github/workflows/release-executor.yml

```
@@ -0,0 +1,115 @@
+        with:
+          path: artifacts
+          merge-multiple: false
+
+      - name: List artifacts
+        run: |
+          ls -la artifacts/
+          find artifacts -type f
+
+      - name: Get release tag
+        id: tag
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:2:54: Double quote to prevent globbing and word splitting

(shellcheck)